spec/index.bs

```diff
@@ -537,7 +537,7 @@ dictionary GPUPipelineStageDescriptor {
 };
 
 dictionary GPUPipelineDescriptorBase {
-    required GPUPipelineLayout layout;
+    GPUPipelineLayout? layout = null;
```

Comment:
It's a dictionary, so I'm not sure it's valuable to say `?`

Reply:
Yeah, if it's not `required`, it's implicitly optional. I think you can just write this as `GPUPipelineLayout layout;` (the [checker](https://w3c.github.io/webidl2.js/checker/) says this is ok).